packages/@aws-cdk/aws-appmesh/lib/virtual-service.ts

```diff
@@ -72,16 +63,6 @@ export interface VirtualServiceBaseProps {
   readonly clientPolicy?: ClientPolicy;
 }
 
-/**
- * The properties applied to the VirtualService being define
- */
-export interface VirtualServiceProps extends VirtualServiceBaseProps {
-  /**
-   * The Mesh which the VirtualService belongs to
-   */
-  readonly mesh: IMesh;
-}
-
```

Comment:
Like I wrote above in `mesh.ts`, I would revert all of these changes, keep `VirtualServiceBaseProps`, and change `readonly mesh: IMesh` in `VirtualServiceProps` to `readonly virtualServiceProvider: VirtualServiceProvider`.

Reply:
Done!